routes/evaluateSubmission.js

```diff
@@ -5,32 +5,44 @@ const Users = require("../models/user.model");
 const FinalProjectAssignments = require("../models/finalProjectAssignment.model");
 
 evaluateSubmission.post("/toEvaluateSubmission", async (req, res) => {
-  const { empId, score, feedback, show, gradedBy } = req.body;
-  const users = await Users.findOne({ empId });
-
-  await FinalProjectAssignments.updateOne(
-    { userId: users?._id, status: true },
-    { status: false }
-  );
-  let data = {
-    projectScore: score,
-    feedback: feedback,
-    status: true,
-    show: show,
-    gradedOn: Date.now(),
-    gradedBy,
-  };
-  //find and update
-  let updated = await FinalProjectAssignments.updateOne(
-    {
-      userId: users?._id,
-      status: false,
-    },
-    data
-  );
-  // sending updating status to forntend
-  if (updated?.modifiedCount === 1) res.json(true);
-  else res.json(false);
+  try {
+    const { empId, score, feedback, show, gradedBy } = req.body;
+    const users = await Users.findOne({ empId });
+    //if user is not found
+    if (!users) {
+      return res.status(404).json({ message: "User not found" });
+    }
+    await FinalProjectAssignments.updateOne(
+      { userId: users?._id, status: true },
+      { status: false }
+    );
+    // if submission is not found
```

Comment:
please this lines of code
 if (!updatedSubmission) {
      return res.status(404).json({ message: "Submission not found" });
    }